app/routes/events/components/JoinEventForm.tsx

```diff
@@ -477,7 +476,6 @@ const JoinEventForm = (props: Props) => {
                       component={TextInput.Field}
                       label={feedbackLabel}
                       className={styles.feedbackText}
-                      fieldClassName={styles.feedbackField}
```

Comment:
Did the css class not exist, or did you forget to remove it? Or is it used anywhere else? I'm on my phone so I can't check sorry

Reply:
yeah it doesn't exist. TS gives you an error when you use styles that don't exist :))